kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/spec/ContainerSpec.java

```diff
@@ -44,6 +45,13 @@ public ReleaseId getReleasedId() {
     public ContainerSpec() {
     }
 
+    public ContainerSpec(ContainerSpec other) {
+        super(other.getId(), other.getContainerName(), other.getServerTemplateKey());
+        this.releasedId = other.releasedId;
+        this.status = other.status;
+        this.configs = other.getConfigs().isEmpty() ? new EnumMap<Capability, ContainerConfig>(Capability.class) : new EnumMap<>(other.getConfigs());
```

Comment:
Here we use `EnumMap` but the initializer is set to `HashMap`, see:
https://github.com/kiegroup/droolsjbpm-integration/blob/1b42ad1f1009442f13a500842c97b16e19d06648/kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/spec/ContainerSpec.java#L36